changelog.md

```diff
@@ -1,3 +1,5 @@
+2024-10-25 - a7f234223a3a34e55a77e3c216733c9f01e5e3c9 -TAR-1977 Preboarding keyboard accessibility issue fixes
```

Comment:
This need to define what change you have done in what file. 

directives/focus-first-element/index.ts - Update the focus first directive to wait for animations to stop

Reply:
changed